source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/notifications/data.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { withData as withOrbit, WithDataProps } from 'react-orbitjs';
+import { TYPE_NAME as NOTIFICATION, NotificationAttributes } from '@data/models/notification';
+import { query, defaultSourceOptions } from '@data';
+import { withStubbedDevData } from '@data/with-stubbed-dev-data';
+import { uuid } from '@orbit/utils';
+
+// const mapNetworkToProps = (passedProps) => {
+//   return {
+//     notifications: q => q.findRecords(NOTIFICATION)
+//   };
+// };
+
+const mapRecordsToProps = (passedProps) => {
+  return {
+    notifications: q => q.findRecords(NOTIFICATION)
+  };
+};
+
+export interface DataProps {
+  notifications: Array<JSONAPI<NotificationAttributes>>;
+  seenNotifications: Array<JSONAPI<NotificationAttributes>>;
+  showNotifications: Array<JSONAPI<NotificationAttributes>>;
```

Comment:
show notifications?

what does this represent?

Reply:
I'm bad a naming things but I use does two variable to check if I need to display red dot (all seen or not) and the other one if they need to be displayed or not (when you clear all or x)

Reply:
does it represent _all_ notifications? or just a subset?